packages/contracts/test/src/token/CBLTest.t.sol

```diff
@@ -25,11 +26,16 @@ contract CBLTest is Test {
         cbl = deployCBLToken.runTest();
     }
 
+    function test__CBL__ShouldRevertOnZeroAddress() public {
+        vm.expectRevert(CBL.CBL__ZeroAddress.selector);
+        new CBL(address(0), address(0), type(uint32).max);
```

Comment:
1. This single test case does not prove that ZeroAddress happens when both parameters are `address(0)`. It only proves it for the first one checked. 
2. We changed our approach to have specific errors for zero address errors.